contracts-v1/deploy/7000_contracts_setup.ts

```diff
@@ -30,7 +30,21 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     await proposals.connect(TestFoundation).setEpochs(epochs.address);
 
     const target: WithdrawalsTarget = await hre.ethers.getContract(WITHDRAWALS_TARGET);
-    await TestFoundation.sendTransaction({ to: target.address, value: parseEther('0.01') });
+
+    if (['sepolia'].includes(hre.network.name)) {
```

Comment:
I'm not sure if it helps, transactions after this change are failing with `out of gas` - https://sepolia.etherscan.io/tx/0x1def4068c7fe5cf7de35dfe8aa0ccaa57dd8524b16196d0b596d6cf1b67c4fa4
But I checked it again from `develop` and now when deploying locally this issue seems to be gone and it works. Could you check if it still occurs when we're deploying from github actions?

Reply:
Indeed, the `sendTransaction` wasn't waiting for confirmations. I've added await for at least 1 confirmation as well as increased gas limit to `50_000`

Reply:
I agree that we can wait for a confirmation, so we know when the full deployment is done. But I still don't think that changing `gasLimit` and `type` will help. `type` is 2 by default on Sepolia and `gasLimit` will set automatically in case of just a normal transfer.  

We might set higher priority fee (it's 1,5 GWEI by default) and higher fee per gas.
```
maxFeePerGas: 500_000000000,
maxPriorityFeePerGas: 3_000000000,
```

Reply:
I've set these values, but I don't know how to ensure that this issue is resolved until it occurs again